_slides/2019-05-28-no-72-electing-hacks.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: slides
+title: Berlin Hack & Tell \#72 - Black hole & Tell
```

Comment:
Change title to "Electing Hacks" . Otherwise the Slides link will not appear on the hacks page (I match the front matter titles).